outport/drivers/interface.go

```diff
@@ -0,0 +1,22 @@
+package drivers
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core/statistics"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/outport/types"
+)
+
+// Driver is an interface for saving node specific data to other storage.
+// This could be an elastic search index, a MySql database or any other external services.
+type Driver interface {
```

Comment:
Why do we have this interface here? Shouldn't we move it directly in /outport package where we defined the `OutportHandler` interface?

Reply:
done. interface moved.